libraries/AP_HAL_ChibiOS/hwdef/CubeRedPrimary/hwdef.dat

```diff
@@ -287,10 +287,6 @@ SERIAL_ORDER OTG1 USART2 USART6 USART3 UART4 UART8 OTG2 UART7
 EXT_FLASH_SIZE_MB 32
 INT_FLASH_PRIMARY 1
 
-# set protocol for UART7 to SerialProtocol_PPP
-define DEFAULT_SERIAL7_PROTOCOL SerialProtocol_PPP
-define DEFAULT_SERIAL7_BAUD 8000000
```

Comment:
These conflict with setting UART7 to IOMCU on the next line.

Reply:
No it doesn't this is a default parameter if IOMCU is disabled. This is a necessary change.

Reply:
removed